pkgs/development/libraries/agda/functional-linear-algebra/default.nix

```diff
@@ -10,16 +10,15 @@ mkDerivation rec {
     repo = "functional-linear-algebra";
     owner = "ryanorendorff";
     rev = "v${version}";
-    sha256 = "032gl35x1qzaigc3hbg9dc40zr0nyjld175cb9m8b15rlz9xzjn2";
+    sha256 = "05jk3792k9xf8iiwzm2hwlvd25f2pqqr3gppmqjf8xb9199i8fk0";
   };
 
   preConfigure = ''
     sh generate-everything.sh
   '';
 
   meta = with lib; {
-    # Remove if a version compatible with agda 2.6.2 is made
-    broken = true;
+    broken = false;
```

Comment:
You can simply remove this line, as `broken` only needs to be set when it is `true`.